src/cosmosdb-preview/azext_cosmosdb_preview/_params.py

```diff
@@ -139,6 +139,55 @@ def load_arguments(self, _):
         c.argument('account_name', account_name_type, id_part=None)
         c.argument('role_assignment_id', options_list=['--role-assignment-id', '-i'], validator=validate_role_assignment_id, help="Optional for Create. Unique ID for the Role Assignment. If not provided, a new GUID will be used.")
         c.argument('role_definition_id', options_list=['--role-definition-id', '-d'], validator=validate_fully_qualified_role_definition_id, help="Unique ID of the Role Definition that this Role Assignment refers to.")
-        c.argument('role_definition_name', options_list=['--role-definition-name', '-n'], help="Unique Name of the Role Definition that this Role Assignment refers to. Eg. 'Contoso Reader Role'.")
         c.argument('scope', validator=validate_scope, options_list=['--scope', '-s'], help="Data plane resource path at which this Role Assignment is being granted.")
         c.argument('principal_id', options_list=['--principal-id', '-p'], help="AAD Object ID of the principal to which this Role Assignment is being granted.")
+
+    # Managed Cassandra Cluster
+    with self.argument_context('cassandra-mi cluster create') as c:
+        c.argument('location', options_list=['--location', '-l'], help="Location", required=True)
+        c.argument('cluster_name', options_list=['--cluster-name', '-c'], help="Cluster Name", required=True)
+        c.argument('delegated_management_subnet_id', options_list=['--delegated-management-subnet-id', '-s'], help="Delegated SubnetId", required=True)
+
+    # Managed Cassandra Cluster
+    with self.argument_context('cassandra-mi cluster update') as c:
+        c.argument('cluster_name', options_list=['--cluster-name', '-c'], help="Cluster Name", required=True)
+
+    # Managed Cassandra Cluster
+    with self.argument_context('cassandra-mi cluster node status') as c:
```

Comment:
commands should look like name-of-some-object verb

in should be something like:  cassandra-cluster-node-status show
Is there a way to make cassandra-cluster-node-status shorter ?

Reply:
It has to be cluster node-status.

Reply:
@evelyn-ys any recommendations for the command name here. To me it looks like we are not using a verb, which is not correct. 

---
In reply to: [582376886](https://github.com/Azure/azure-cli-extensions/pull/3035#discussion_r582376886) [](ancestors = 582376886)